### ISSUE TITLE
DNS SRV lookup problem

### ISSUE THREAD

**Issue by yalosev** [telepresenceio/telepresence#435]
I have found some confusing dns srv lookup behavior with FQDN.
If you are looking for A records, everything works fine:
```
bash-4.3$ dig elasticsearch
 
;; ANSWER SECTION:
elasticsearch.		0	IN	A	10.0.0.139
```
```
bash-4.3$ dig elasticsearch.default.svc.cluster.local

;; ANSWER SECTION:
elasticsearch.default.svc.cluster.local. 0 IN	A 10.0.0.139
```

But when you use srv lookup, it becomes confusing.
Sure srv port is described in service config:

>   ports:
>   - name: http
>     port: 9200
>     protocol: TCP
>     targetPort: elasticsearch

First example looks fine:
```
bash-4.3$ dig srv _http._tcp.elasticsearch.default.svc.cluster.local

;; ANSWER SECTION:
_http._tcp.elasticsearch.default.svc.cluster.local. 30 IN SRV	10 100 9200 elasticsearch.default.svc.cluster.local.

;; ADDITIONAL SECTION:
elasticsearch.default.svc.cluster.local. 30 IN A 10.0.0.139
```

BUT:
```
bash-4.3$ dig srv _http._tcp.elasticsearch

; <<>> DiG 9.10.3-P4-Ubuntu <<>> srv _http._tcp.elasticsearch
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 58299
;; flags: qr ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 0

;; QUESTION SECTION:
;_http._tcp.elasticsearch.	IN	SRV

;; Query time: 58 msec
;; SERVER: 127.0.0.1#53(127.0.0.1)
;; WHEN: Mon Jan 29 11:54:42 SAMT 2018
;; MSG SIZE  rcvd: 42
```
Adding `search default.svc.cluster.local` into /etc/resolv.conf doesn`t help.

Maybe I can handle this somehow? Or it`s a bug?

Thank you.

**Comment by donnyyung** [comment]
Not sure if this is still a TP2 issue, so marking as tp2 for now until we confirm  it is/ is not an issue.

**Comment by onmomo** [comment]
@donnyyung yes, it seems not implemented (https://github.com/telepresenceio/telepresence/issues/1863)

**Comment by thallgren** [comment]
Implemented in #2787.
